miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplayTest.kt

```diff
@@ -17,17 +20,19 @@ import kotlin.test.assertTrue
 class RealMiniAppDisplayTest {
 
     lateinit var context: Context
+    lateinit var baseTextPath: String
 
     @Before
     fun setup() {
         context = getApplicationContext()
+        baseTextPath = context.filesDir.absolutePath
```

Comment:
baseTextPath?

Reply:
Please check if you require `absolutePath`.

Reply:
Both `absolutePath` and `path` are fine. Let's change it to `path` for code consistence.